### PR TITLE
Extending sites

### PR DESCRIPTION
**Pull request by h20190563**
Have added five websites to the current pool of websites. Below is the information for the added websites:

HackerEarth: The competitive coding platform where we can practice coding from various domains.
GateOverflow: The resource portal that helps the students by solving their doubts related to the computer science field. This is one of the important sites for the GATE (Entrance exam for pursuing masters in India) preparing students.
GeeksForGeeks: One of the most famous website which contains a large pool of content of various domains in the field of computer science.
NitroType: Gaming webiste
YourQuote: This website provides a platform for writers to showcase their skills. 

All five sites are checked and they are giving proper results.

### CODE REVIEW

File: sherlock/resources/data.json [hunk 1]
```diff
@@ -1,2218 +1,2241 @@
...
-  "uid": {
-    "errorType": "status_code",
-    "url": "http://uid.me/{}",
-    "urlMain": "https://uid.me/",
-    "username_claimed": "blue",
-    "username_unclaimed": "noonewouldeverusethis7"
-  }
-}
\ No newline at end of file
+{
```
Comment (pkubiak) [inline sherlock/resources/data.json:1]: You have changed line endings into `\r\n` in whole file

Comment (h20190563) [reply sherlock/resources/data.json:1]: I didn't get that. Can you please elaborate it a bit?

**Review by Diego199069 — APPROVED** [review]
Very well done

File: sherlock/resources/data.json (hunk 1 above)
Comment (pkubiak) [reply sherlock/resources/data.json:1]: Sure, https://en.wikipedia.org/wiki/Newline#Representation.

I suppose that you have accidentally changed new lines style from `\n` (unix style) to `\r\n` (windows style) in `data.json` file. 

That is indicated by overall changes count (~top right corner of website):  
- you have inserted +2,547 lines
- you have removed −2,512 lines
And in `Files Changed` tab you will see that every line of that files has been changed.

Comment (h20190563) [reply sherlock/resources/data.json:1]: Thanks for explaining it in detail.

**Comment by h20190563** [comment]
@pkubiak  Is the format okay now? I have also added the support of YourQuotes.

File: sherlock/resources/data.json (hunk 1 above)
Comment (h20190563) [reply sherlock/resources/data.json]: @pkubiak  Is the format okay now? I have also added the support of YourQuotes.

**Pull request closed without merging**